python/pyspark/sql/dataframe.py

```diff
@@ -358,22 +360,19 @@ def show(self, n=20, truncate=True, vertical=False):
     def _eager_eval(self):
```

Comment:
If this can be replaced by sql_conf access, we don't really need this private function.

Reply:
Yep, agree.

Reply:
Thanks, Done in 453004a.